arabic/journey/journey to the unknown.md

```diff
@@ -0,0 +1 @@
+كانت رحلة جميلة قضيتها مع زوجتى فى احدى المنتجعات الرائعة وبينما نحن فى طريقنا للعودة ليلا : اذ يالعربة تتعطل فجأة
```

Comment:
i need to add a new directory has a new page to arabic how to do so ??
@cbuckey-uda